gazebo_plugins/CMakeLists.txt

```diff
@@ -12,11 +12,44 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(ament_cmake REQUIRED)
-find_package(gazebo_ros REQUIRED)
 find_package(gazebo_dev REQUIRED)
+find_package(gazebo_ros REQUIRED)
+find_package(rclcpp REQUIRED)
+
+include_directories(include
+  ${gazebo_ros_INCLUDE_DIRS}
+)
+
+# gazebo_ros_template
+add_library(gazebo_ros_template SHARED
+  src/gazebo_ros_template.cpp
+)
+target_link_libraries(gazebo_ros_template
+  ${gazebo_ros_LIBRARIES}
+)
```

Comment:
I believe this is handled by ```ament_target_dependencies```.

Reply:
You're right, it seems to work without it. I wonder what I was doing before that I needed to add this. Removing for now: https://github.com/ros-simulation/gazebo_ros_pkgs/pull/783/commits/c86ee0cd8b77cf161724f4c1bc78419ef62c3694